src/plugins/I18nPlugin.ts

```diff
@@ -137,14 +112,65 @@ export default class DojoI18nPlugin {
 			});
 		}
 
-		compiler.plugin('compilation', (compilation: any) => {
+		compiler.plugin('compilation', (compilation: any, data: any) => {
+			const astMap = new Map();
+			const containsLoad: string[] = [];
+
+			data.normalModuleFactory.plugin('before-resolve', (result: any, callback: any) => {
+				if (!result) {
+					return callback();
+				}
+
+				let request = isRelative(result.request) ? path.join(result.context, result.request) : result.request;
+
+				const { contextInfo } = result;
+				const issuer = contextInfo && contextInfo.issuer;
+				if (issuer && issuer.indexOf('node_modules') < 0 && isCldrLoadModule(request)) {
+					containsLoad.push(issuer);
+				}
+
+				return callback(null, result);
+			});
+
+			data.normalModuleFactory.plugin('parser', (parser: any) => {
+				parser.plugin('program', (ast: any) => {
+					const path = parser.state.current.userRequest;
+					if (path) {
+						astMap.set(path, ast);
```

Comment:
Actually, if I map dependency mids to their issuers during the `before-resolve` event, and then remove them during the `parser` event, I should be able to track when all dependencies have been parsed.